src/components/label/template.njk

```diff
@@ -1,14 +1,15 @@
 {% from "error-message/macro.njk" import govukErrorMessage -%}
 
-<label class="govuk-c-label
-  {%- if params.classes %} {{ params.classes }}{% endif %}" for="{{ params.id }}">
-  {{ params.labelText }}
+<label class="govuk-c-label{%- if params.classes %} {{ params.classes }}{% endif %}"
+  {%- for attribute, value in params.attributes %} {{attribute}}="{{value}}"{% endfor %}
+  {%- if params.for %} for="{{ params.for }}"{% endif %}>
+  {{ params.html | safe if params.html else params.text }}
   {% if params.hintText %}
   <span class="govuk-c-label__hint">
     {{ params.hintText }}
   </span>
   {% endif %}
   {% if params.errorMessage %}
-  {{ govukErrorMessage(params.errorMessage) -}}
+  {{ govukErrorMessage(params.errorMessage.text) -}}
```

Comment:
This should just be params.errorMessage so that you can pass `html` through etc.

Reply:
Yep, already fixed that :P